src/app/submitted/submitted.component.ts

```diff
@@ -0,0 +1,17 @@
+import { Component, OnInit } from '@angular/core';
+import { Router } from '@angular/router';
+
+@Component({
+  selector: 'app-submitted',
+  templateUrl: './submitted.component.html',
+  styleUrls: ['./submitted.component.scss'],
+})
+export class SubmittedComponent implements OnInit {
+  constructor(private router: Router) {}
+
+  ngOnInit() {
+    setTimeout(() => {
+      this.router.navigate(['']);
+    }, 6000);
```

Comment:
I thought that it would be nice if after submitting form and navigated to the "thank you" page we would only "stay" there for a bit and then would be navigated back to the home page. But of course I can remove it.